services/brig/src/Brig/User/Search/Index.hs

```diff
@@ -267,14 +269,23 @@ createIndex' ::
   [ES.UpdatableIndexSetting] ->
   -- | Number of shards
   Int ->
+  Maybe ES.TemplateName ->
   m ()
-createIndex' failIfExists settings shardCount = liftIndexIO $ do
+createIndex' failIfExists settings shardCount mbTemplate = liftIndexIO $ do
   idx <- asks idxName
   ex <- ES.indexExists idx
   when (failIfExists && ex) $
     throwM (IndexError "Index already exists.")
   unless ex $ do
     let fullSettings = settings ++ [ES.AnalysisSetting analysisSettings]
+
```

Comment:
Can you please add a comment here explaining why we do this and maybe drop a link to an issue which will delete this code?

Reply:
Done